edx-platform/pearson-vue-theme/lms/templates/courseware/info.html

```diff
@@ -0,0 +1,122 @@
+<%page expression_filter="h"/>
```

Comment:
What are the custom changes to this template? @sergivalero20

Reply:
@Squirrel18  In this template only was changed some texts in several links and it was removed an element related to page-subtitle [commit](https://github.com/Pearson-Advance/openedx-themes/commit/181ab602d532a2a20d64dec68cd712205dbb4ba2#diff-6cc945d0e7aa824263069b6e9d6ab052a9f955160a53d08e02378303ebe311c6).

Reply:
Info.html doesn't appear in Olive Version.

Reply:
Same comment as before. @sergivalero20